connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Loggers.java

```diff
@@ -179,17 +178,23 @@ private synchronized List<org.apache.logging.log4j.Logger> loggers(String namesp
 
     // visible for testing
     org.apache.logging.log4j.Logger lookupLogger(String logger) {
-        return LogManager.getLogger(logger);
+        return LogManager.getLogger(isValidRootLoggerName(logger) ? LogManager.ROOT_LOGGER_NAME : logger);
     }
 
     List<org.apache.logging.log4j.Logger> currentLoggers() {
         LoggerContext context = (LoggerContext) LogManager.getContext(false);
-        Collection<LoggerConfig> loggerConfigs = context.getConfiguration().getLoggers().values();
-        return loggerConfigs.stream()
+        Set<String> loggers = context.getLoggerRegistry().getLoggers()
```

Comment:
it seems we can return the loggers directly:
```java
        LoggerContext context = (LoggerContext) LogManager.getContext(false);
        return context.getLoggerRegistry().getLoggers()
                .stream()
                .map(r -> (org.apache.logging.log4j.Logger) r)
                .toList();
```

Reply:
Additionally, the API is expected to return root logger, so we can call `context.getRootLogger()` to ensure the root logger is created.